src/cli/commands/deposit.ts

```diff
@@ -18,31 +26,32 @@ export class DepositCommand implements CliCommand {
       .option("-n, --node [node]", 'Url of eth1 node', 'http://127.0.0.1:8545')
       .option("-v, --value [value]", 'Amount of ether to deposit', "32")
       .option("-c, --contract [contract]", 'Address of deposit contract', defaults.depositContract.address)
-      .action( async ({privateKey, mnemonic, node, value, contract}) => {
-        //library is not awaiting this method so don't allow error propagation (unhandled promise rejections
+      .action( async (options) => {
+        //library is not awaiting this method so don't allow error propagation 
+        // (unhandled promise rejections)
         try {
-          await this.action(privateKey, mnemonic, node, value, contract);
+          await this.action(options);
         } catch (e) {
           logger.error(e.message);
         }
 
       });
   }
 
-  public async action(privateKey: string, mnemonic: string, node: string, value: string, contract: string) {
-    const provider = new ethers.providers.JsonRpcProvider(node);
+  public async action(options: IDepositCommandOptions): Promise<void> {
+    const provider = new ethers.providers.JsonRpcProvider(options.node);
     try {
       //check if we can connect to node
       await provider.getBlockNumber();
     } catch (e) {
-      throw new CliError(`JSON RPC node (${node}) not available. Reason: ${e.message}`);
+      throw new CliError(`JSON RPC node (${options.node}) not available. Reason: ${e.message}`);
     }
 
     const wallets = [];
-    if(mnemonic) {
-      wallets.push(...this.fromMnemonic(mnemonic, provider, 10));
-    } else if (privateKey) {
-      wallets.push(new Wallet(privateKey, provider));
+    if(options.mnemonic) {
+      wallets.push(...this.fromMnemonic(options.mnemonic, provider, 10));
```

Comment:
Since you're fixing the mnemonic, we should allow an override for `n` number of wallets and default it int he flag to 10. Something like:
`option.( -a, --accounts, "Number of accounts to generate at startup", 10)`

Reply:
will do !

Reply:
Add this default to defaults and save it with create-config, or is it not important?